src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNITcpHandle.cs

```diff
@@ -409,20 +442,21 @@ void Cancel()
                                         sockets[n] = null;
                                     }
                                 }
-                                n++;
                             }
                         }
                         catch (Exception e)
                         {
                             SqlClientEventSource.Log.TrySNITraceEvent(s_className, EventType.ERR, "THIS EXCEPTION IS BEING SWALLOWED: {0}", args0: e?.Message);
+                            SqlClientEventSource.Log.TryAdvancedTraceEvent($"{s_className}.{System.Reflection.MethodBase.GetCurrentMethod().Name}{EventType.ERR}THIS EXCEPTION IS BEING SWALLOWED: {e}");
```

Comment:
I'm going to have to use the static values to avoid including more changes in this PR. We should modify `TryAdvancedTraceEvent` method like `TrySNITraceEvent`.